OpenFGA.g4

```diff
@@ -33,11 +33,8 @@ relationDefTypeRestrictionRelation: name;
 relationDefTypeRestrictionWildcard: relationDefTypeRestrictionType ':*';
 relationDefTypeRestrictionUserset: relationDefTypeRestrictionType '#' relationDefTypeRestrictionRelation;
 
-//relationDefComplexGrouping: relationDefGroup (spacing relationDefOperator spacing relationDefGroup)*;
-//relationDefGroup: '(' spacing? (relationDefOrGrouping | relationDefAndGrouping | relationDefButNotGrouping) spacing? ')';
-//relationDefAndGrouping: relationDefRewrite spacing relationDefOperatorAnd (spacing relationDefAndGrouping)+;
-//relationDefOrGrouping: relationDefRewrite spacing relationDefOperatorOr (spacing relationDefOrGrouping)+;
-//relationDefButNotGrouping: relationDefRewrite spacing relationDefOperatorButNot (spacing relationDefButNotGrouping)+;
+relationDefGrouping: relationDefRewrite;
+relationDefGroup: '('  relationDefGrouping relationDefPartials* ')';
```

Comment:
Do we now allow ( XXX ) in our language?  If so, do we want to call it out in the PR?

Reply:
Not yet, this part is there but unused

Reply:
Removed now, to be readded later